Source/Icebreaker/Services/MatchingService.cs

```diff
@@ -185,12 +197,51 @@ private string GetChannelUserObjectId(ChannelAccount account)
             return JObject.FromObject(account).ToObject<TeamsChannelAccount>()?.AadObjectId;
         }
 
+        /// <summary>
+        /// Parse through a list of pairing information.
+        /// </summary>
+        /// <param name="pairHistory">List of pairing information.</param>
+        /// <returns>Returns pairs from the most recent pairing phase. This can be changed to include
```

Comment:
Please be more specific as to _what_ the return structure is. It looks like this is a mapping from a user's id to the set of user ids they matched with in the 'last' iteration.